CryptoTechReminderSystem.Test/Gateway/HarvestGatewayTests.cs

```diff
@@ -39,7 +39,7 @@ public void TearDown()
             private void SetUpUsersApiEndpoint(string id, string firstName, string lastName, string email)
             {
                 var json = $"{{  \"users\":[    {{      \"id\":{id},      \"first_name\":\"{firstName}\"" +
```

Comment:
is there a need for this as we have a json file now?

Reply:
will be sorted in a separate PR